package.json

```diff
@@ -64,6 +64,7 @@
     "@react-navigation/core": "^3.5.1",
     "@react-navigation/native": "^5.0.9",
     "@react-navigation/stack": "^5.1.1",
+    "abortcontroller-polyfill": "^1.4.0",
```

Comment:
I thought `AbortController` was added to RN already:

https://github.com/facebook/react-native/commit/5e36b0c6eb2494cefd11907673aa018831526750 

Looks like it's just a global?

Reply:
5128a3a.